### PR TITLE
deps: bump person-directory to 1.8.16

### PR DESCRIPTION
**Pull request by jgribonvald**
##### Checklist

-   [X] the [individual contributor license agreement][] is signed
-   [X] commit message follows [commit guidelines][]
-   [ ] tests are included
-   [ ] documentation is changed or added
-   [ ] [message properties][] have been updated with new phrases
-   [ ] view conforms with [WCAG 2.0 AA][]

##### Description of change

Update person-directory dependency with new IPersonAttributeDao methods signature (would have been a breaking change in dependency).

Warning: only applies compatibility change, the new lib introduced a Filter parameter, so in some case a local implementation of the filter could be a better way than filtering after methods call with a old java code.

[individual contributor license agreement]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#individual-contributor-license-agreement
[commit guidelines]: https://github.com/Jasig/uPortal/blob/master/CONTRIBUTING.md#commit
[message properties]: https://github.com/Jasig/uPortal/tree/master/uportal-war/src/main/resources/properties/i18n
[WCAG 2.0 AA]: https://www.w3.org/WAI/WCAG20/quickref/?levels=aaa&technologies=smil%2Cpdf%2Cflash%2Csl

### CODE REVIEW

**Review by ChristianMurphy — APPROVED** [review]
Does the filter always need to be passed?
I'm a bit surprised it doesn't/wouldn't default to `alwaysChoose`.

Other than that, LGTM, thanks!

**Comment by jgribonvald** [comment]
Any other review before to merge ?

Thanks

**Comment by jgribonvald** [comment]
Finally don't merge it, I don't know why my test didn't show me a problem when using it in pre-production, but it seems that the new version need some cache configuration as it provides several warn due to that.
So don't know, maybe the best is to wait to upgrade to latest version when migrating to next major version ?

**Pull request closed without merging**
